src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String path) {
+        List<String> operations;
+        try {
+            operations = Files.readAllLines(Path.of(path));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file in: " + path, e);
+        }
+        return operations.subList(1, operations.size());

```

Comment:
Skipping the first line of the file assumes that it is always a header. It would be more robust to check if the first line is a header and skip it accordingly, or to handle this logic outside of the `FileReaderImpl` class to make it more reusable.